clang/test/CodeGenSYCL/esimd-accessor-ptr-md.cpp

```diff
@@ -1,21 +1,10 @@
-// TODO: previously code generation and ESIMD lowering was
-// a part of the same %clang_cc1 invocation, but now it is
-// separate. So, we can split this test into 2, where one
-// will be testing code generation and the second ESIMD lowering.
-//
 // RUN: %clang_cc1 -fsycl-is-device \
 // RUN:   -internal-isystem %S/Inputs -triple spir64-unknown-unknown-sycldevice \
-// RUN:   -disable-llvm-passes -emit-llvm %s -o %t
-// RUN: sycl-post-link -split-esimd -lower-esimd -O0 -S %t -o %t.table
-// RUN: FileCheck %s -input-file=%t_esimd_0.ll
+// RUN:   -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
 
 // This test checks
-// 1) proper metadata generation for accessors used in ESIMD
-// kernels:
-//   - Proper 'kernel_arg_accessor_ptr' metadata is generated by the FE for
-//     ESIMD kernels
-//   - Pointers originating from accessors are marked with 'buffer_t' and proper
```

Comment:
Are we just simply removing the checks from here w/o reproducing them elsewhere (e.g. in new LowerESIMD tests)?

Reply:
No, there are existing tests on LowerESIMD pass that check that. I mentioned this in the commit message.

Reply:
ok, I see, thanks.